### PR TITLE
Bouncy icon

### PR DESCRIPTION
**Pull request by jamesgorrie**
## What is this PR trying to achieve?
Fixes #624

## What does it look like?
bounce

### CODE REVIEW

**Comment by Norvard** [comment]
Wicked!!

File: client/scss/components/_find_us.scss [hunk 1]
```diff
@@ -40,7 +40,8 @@
     color: color('action-green-2');
 
     .icon {
-      top: -0.2em;
+      animation-duration: 400ms;
+      animation-name: find-us-icon-bounce;
```
Comment (davidpmccormick) [inline client/scss/components/_find_us.scss]: We could/should we add this animation for all the social icons as well?

Comment (Norvard) [inline client/scss/components/_find_us.scss]: yep that would be cool!

**Comment by jamesgorrie** [comment]
>  if we should use this animation everywhere

Power vs responsibility... we need to start measuring performance before going headfirst into this.

**Pull request merged**